package.json

```diff
@@ -108,8 +108,8 @@
     "browserslist-config-design-italia": "^1.0.0",
     "chromatic": "^6.3.4",
     "circular-dependency-plugin": "^5.2.2",
-    "eslint-plugin-mdx": "^1.13.0",
-    "eslint-plugin-prettier": "^3.4.0",
+    "eslint-plugin-mdx": "1.17.1",
+    "eslint-plugin-prettier": "^4.2.1",
     "husky": "^7.0.4",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^26",
```

Comment:
### Jest changes

To make test pass it requires a little bit more effort than planned.
Resolutions didn't work well for the `jest` upgrade, so we have to override it here:

```suggestion
    "jest": "27.5.1"
```

And also the `ts-jest` below:

```
"ts-jest": "^27.1.5",
```

and add `"babel-jest"` as newer dep:

```
"babel-jest": "^27.5.1",
```

In the script section replace `tsdx test` with `npx jest` for both `"test"` and `"test:ci"` scripts.

In the "jest" configuration add `jsdom` as environment (it was a breaking change in jest@27):
```
"testEnvironment": "jsdom",
```

In the `jest-transformer.js` file add a `.default` to the `babel-jest` require:

```
const babelJest = require('babel-jest').default;
```